azurerm/config.go

```diff
@@ -247,7 +247,7 @@ func getArmClient(authConfig *authentication.Config, skipProviderRegistration bo
 	client.monitor = monitor.BuildClient(o)
 	client.mssql = mssql.BuildClient(o)
 	client.msi = msi.BuildClient(o)
-	client.mysql = mysql.BuildClient(o)
+	client.mssql = mssql.BuildClient(o)
```

Comment:
this is the same line as 249?

Reply:
Yes, meanwhile someone else fixed this on master already. I'll remove this again.